backend/src/api/routers/graphs.py

```diff
@@ -0,0 +1,71 @@
+import os
+
+from fastapi import APIRouter
+from pydantic import BaseModel
+
+from ...agents.retriever_graph import RetrieverGraph
+
+from langchain_google_vertexai import ChatVertexAI
+from langchain_google_genai import ChatGoogleGenerativeAI
+
+from dotenv import load_dotenv
+
+
+class UserInput(BaseModel):
+    query: str
+    list_sources: bool = False
+    list_context: bool = False
+
+
+load_dotenv()
+
+use_cuda = os.getenv("USE_CUDA")
+
+if os.getenv("GOOGLE_GEMINI") == "1_pro":
+    llm = ChatGoogleGenerativeAI(model="gemini-pro", temperature=1)
```

Comment:
Can we make `temperature` a default? Then we don't have to specify it here. Also why are you specifying only for gemini-pro and not the others?

Reply:
I would be concerned by the LLM responses based on a high temperature setting in the code given the randomness this could generate.

Reply:
1 is the default value. `temperature` is an env variable now.